docs/src/main/asciidoc/README.adoc

```diff
@@ -0,0 +1,238 @@
+:repo_url: https://github.com/marcingrzejszczak/database-update
+
+= Zero Downtime Deployment Database Issues
+
+{repo_url}
+
+This article will explain in more depth how to tackle issues related to database compatibility with Spring Boot and
+database versioning tooling. We will present what can happen with your production applications if you try to perform
+such a deployment unprepared. We will also walk through the lifecycle of an application in order to perform zero
+downtime deployment and at the end of day apply a backwards incompatible database change in a backwards compatible way.
```

Comment:
"at the end of the day" seems like the wrong idiom here. You probably should start a new sentence and write "And finally, we will see how to ..."? Or something.